matlab_simulation/08-planning/example_search_graph.m

```diff
@@ -0,0 +1,47 @@
+% Get same problem every time (fix random number generator start point)

```

Comment:
"clear;clc" might be added at the beginning of "example_search_graph.m" for a clear and successful start.